drivers/sensor/tdk/icm42688/icm42688_decoder.c

```diff
@@ -305,7 +305,7 @@ static uint32_t accel_period_ns[] = {
 };
 
 static uint32_t gyro_period_ns[] = {
-	[ICM42688_DT_GYRO_ODR_12_5] = UINT32_C(10000000000000) / 12500,
+	[ICM42688_DT_GYRO_ODR_12_5] = UINT32_C(10000000000000) / 125000,
```

Comment:
Is this related to `drivers: icm42688: pin9 function`? It looks like an unrelated bug fix